hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncTableBuilder.java

```diff
@@ -137,6 +137,11 @@ default AsyncTableBuilder<C> setMaxRetries(int maxRetries) {
    */
   AsyncTableBuilder<C> setStartLogErrorsCnt(int startLogErrorsCnt);
 
+  /**
+   * Set a request attribute
```

Comment:
Can you please add a little more detail to this javadoc (as well as on TableBuilder and the getRequestAttributes() methods on AsyncTable and Table)? We should provide enough detail so that people know what these are and how they work.